### DISCUSSION TITLE
Affichage automatique du synoptique en fonction du support utilisé

### DISCUSSION THREAD

**Discussion by ABINolan** [witsa/synapps#679] [Q&A] [answered]
Bonjour,
Je travail sur un projet avec un synoptique mais l'affichage de celui-ci diffère lorsque je bascule d'un écran pc à un écran de tablette.
J'aimerais savoir comment la résolution de mon syno sur wit en px peux s'adapter en fonction de la taille de l'écran utilisé.

Affichage sur écran de PC :
![Capture](https://github.com/witsa/synapps/assets/154320516/a1317af9-85ee-4ba4-a4ff-08c2e39e6ba2)

Affichage sur écran de PC portable :
![Capture](https://github.com/witsa/synapps/assets/154320516/a44f2f1e-0392-4dc6-8908-ffbfc98e76d1)

Merci par avance,
Nolan

**Comment by manufitoussiwit** [comment, marked as answer]
Bonjour,

La clé c'est d'utiliser l'acteur boite à vue qui permet de disposer un plan syno fait en px, de manière à ce qu'il soit toujours bien ajusté.

[Documentation : Boite à vue - usage](https://witsa.github.io/synapps/concepts/actor-types/layout-view-box.html#usage)



> Attention à bien utiliser l'unité px dans ce qu'on va mettre dans l'acteur boite à vue.

Un tuto arrive bientot pour expliquer tout ça :-)


![image](https://github.com/witsa/synapps/assets/35595723/5a10e21d-f2ce-480f-8629-2427fb6c6c69)

**Reply by ABINolan** [reply]
Merci beaucoup

**Comment by edouar1** [comment]
Bonjour, je me permis de répondre a votre discussion, en fait quand on utilise une boite a vue ça redimensionne bien mais on perd le zoom quand en redimensionne au max.
Est ce que y a une solution pour ça svp ?
cordialement,

**Reply by WITNicolas** [reply]
Bonjour,

Si on ne souhaite pas utiliser la boite à vue, il faut alors penser toutes les mesures de positionnement en mode relatif, c'est à dire en utilisant les coordonnées en % par rapport au bords de page (haut/bas/gauche/droite)
Du coup on recrée artificiellement le fonctionnement de la boite à vue, en gardant la possibilité du zoom.

Ne pas oublier que le but de la boite à vue est justement de faire tout rentrer dans la page. Le Zoom n'affiche de par sa fonctionnalité pas toute la page .... Donc 2 fonctionnalités opposées :)

**Comment by edouar1** [comment]
Bonjour, 
Le problème est dés que on enlève la boite à vue en perd le redimensionnement de la page .
Le positionnement on mode relatif est de travailliez qu'avec les % sur les empilements?
Cordialement,

**Reply by WITNicolas** [reply]
Tout a fait ... seulement des coordonnées avec des ratios de taille pour le positionnement, les tailles d'objets, tailles de texte, etc...
Bref recréer tout le responsive design "manuellement".

Ceci dit .... on ne fait jamais cela car beaucoup trop chronophage et complexe pour penser à tous ces mécanismes (je me positionnement en tant qu'utilisateur, même averti, mais pas en tant que développeur WEB)
Le plus simple est de prévoir une taille de type TABLETTE, donc forcément sur PC ça sera affiché correctement.
